sdk/appcenter/src/main/java/com/microsoft/appcenter/ingestion/AppCenterIngestion.java

```diff
@@ -55,6 +51,20 @@
     @VisibleForTesting
     static final String APP_SECRET = "App-Secret";
 
+    /**
+     * Auth token format for Authorization header.
+     */
+    @VisibleForTesting
+    @SuppressWarnings("WeakerAccess")
+    static final String AUTH_TOKEN = "Bearer %s";
```

Comment:
This should be `AUTH_TOKEN_FORMAT`

Reply:
This and `AUTHORIZATION_HEADER` should be in a shared file since they're duplicated [here](https://github.com/Microsoft/AppCenter-SDK-Android/pull/950/files#diff-147e2a59ce7018d183c480e87a829c84R89)